.github/workflows/docker-server.yml

```diff
@@ -7,42 +7,36 @@ on:
         description: "Version of Flower e.g. (1.6.0)."
         required: true
         type: string
-      python-version:
-        description: "Version of Python e.g. (3.11.7 or 3.11). Defaults to the version defined in _docker-server.yaml."
-        required: false
-        type: string
-      pip-version:
-        description: "Version of pip. Defaults to the version defined in _docker-server.yaml."
-        required: false
-        type: string
-      setuptools-version:
-        description: "Version of setuptools. Defaults to the version defined in _docker-server.yaml."
-        required: false
-        type: string
-      ubuntu-version:
-        description: "Version of Ubuntu. Defaults to the version defined in _docker-server.yaml."
+      base-image-version:
+        description: "Version of the Flower base image."
         required: false
         type: string
+        default: "py3.11-ubuntu22.04"
 
 permissions:
   contents: read
 
 jobs:
   build-server-images:
-    uses: ./.github/workflows/_docker-server.yml
+    uses: ./.github/workflows/_docker-build.yml
     with:
-      flwr-version: ${{ github.event.inputs.flwr-version }}
-      python-version: ${{ github.event.inputs.python-version }}
-      pip-version: ${{ github.event.inputs.pip-version }}
-      setuptools-version: ${{ github.event.inputs.setuptools-version }}
-      ubuntu-version: ${{ github.event.inputs.ubuntu-version }}
+      namespace-repository: flwr/server
+      file-dir: src/docker/server
+      build-args: |
+        FLWR_VERSION=${{ github.event.inputs.flwr-version }}
+        BASE_IMAGE_VERSION=${{ github.event.inputs.base-image-version }}
+      tags: |
+        ${{ github.event.inputs.flwr-version }}-${{ github.event.inputs.base-image-version }}
+        ${{ github.event.inputs.flwr-version }}
+        latest
     secrets:
       dockerhub-user: ${{ secrets.DOCKERHUB_USERNAME }}
       dockerhub-token: ${{ secrets.DOCKERHUB_TOKEN }}
 
   summary:
     runs-on: ubuntu-22.04
     needs: build-server-images
+    timeout-minutes: 5
```

Comment:
@Robert-Steiner Should we increase this to 10min to give a bit more time in case the CI is slow due to e.g. bad network speed? I believe 10min is still acceptable for these jobs. Feel free to keep it at 5 if you think it's reasonable.

Reply:
I think we can increase it to 10 to be on the safe side.